go.mod

```diff
@@ -16,10 +16,10 @@ require (
 	github.com/etcd-io/bbolt v1.3.3
 	github.com/ghodss/yaml v0.0.0-20161207003320-04f313413ffd
 	github.com/gogo/protobuf v0.0.0-20170815085658-fcdc5011193f // indirect
-	github.com/gorilla/context v1.1.1 // indirect
-	github.com/gorilla/mux v0.0.0-20170217192616-94e7d24fd285 // indirect
+	github.com/gorilla/context v0.0.0-20140604161150-14f550f51af5 // indirect
+	github.com/gorilla/mux v0.0.0-20140926153814-e444e69cbd2e // indirect
 	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
-	github.com/imdario/mergo v0.0.0-20160517064435-50d4dbd4eb0e
+	github.com/imdario/mergo v0.0.0-20141206190957-6633656539c1
```

Comment:
There are several downgrades here; why are they the right thing to do?

E.g. this one in particular goes over 1 year back; at least some of the others revert the upgrades done in https://github.com/containers/image/pull/682 , which is still a bit suspect but at least obviously motivated, but this import used the 50d4… commit before that PR as well, so this is a straight downgrade.

(I’m not at all saying that the downgrade is incorrect, but it is at least suspect, I can’t take a detailed look _right now_, and there is no explanation in the PR nor the commit message. I’ll try to look into this later myself.)

Reply:
The explanation is ...

> I cannot tell exactly which dependency caused the issue or if it was a bad combination. That's not very satisfying but at least we're back to a working state. Any future updates will catch regressions early which will hopefully help in understanding which dependencies are responsible.

I don't have much time to look into the details but at least wanted to get us back into a working state.

Reply:
> That’s fair, it’s just that this is not AFAICS moving us “back” to c/image as it was. Or is it intended to be “back to c/Skopeo as it was”?

The latter after noticing that the previous ones c/image didn't work either. I used the version that c/skopeo was using since those are the ones used in the CI. The `go.mod` change actually caused c/skopeo to inherit the dependencies.

Reply:
Thanks for checking, @mtrmac !

Reply:
So, based on tests in https://github.com/containers/image/pull/710/commits , this seems to be caused by an `imdario/mergo` upgrade; there’s already a comment pointing to that behavior change in https://github.com/containers/image/blob/ddb11c057155dceb36e44ca82146b2df93b4dd3f/openshift/openshift-copies.go#L209 .

I appreciate that `openshift-copies.go` is a horrible tangle of abstractions (even after cutting down the original a bit); still…

(In the future, please clearly document both what _and why_ has happened, even if the underlying cause is not known; nothing links the the test failures in #724 , nor records them, to this PR, and reverse-engineering the causes for this downgrade would be pretty difficult in a few weeks.)